alphagov/paas-admin
# Bump all dependencies

What
----

This is a naive way, with the use of `ncu -u` updating everything to the latest version.

It's naive, as I am expecting the tests to do most of the lifting and checking if it all works together.

How to review
-------------

- Deploy to your env
- Click around see if everything works as expected
